test/bondingcurve/BondingCurve.test.js

```diff
@@ -270,7 +272,16 @@ describe('BondingCurve', function () {
           });
         });
       });
-        
+      describe('Exceeding cap', function() {
```

Comment:
Can't tell if there is a test in this file that buys under the mint cap to check that its still possible to buy even with the mint cap on

Reply:
To add to this point, I only see purchasing happening in the before each hook and we don't verify the fei they received. Recommend adding a test purchasing from the bonding curve that actually succeeds. After the success, we should verify FEI balances and that the bonding curve or receiving contract's eth balance incremented.

Reply:
There are tests for the FEI being received higher up in the file